src/main/kotlin/com/cognifide/gradle/aem/tooling/tail/LogNotifier.kt

```diff
@@ -0,0 +1,35 @@
+package com.cognifide.gradle.aem.tooling.tail
+
+import com.cognifide.gradle.aem.common.NotifierFacade
+import com.cognifide.gradle.aem.tooling.tail.io.LogFiles
+import java.net.URI
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+import kotlinx.coroutines.channels.ReceiveChannel
+import kotlinx.coroutines.channels.consumeEach
+import kotlinx.coroutines.launch
+
+@UseExperimental(ObsoleteCoroutinesApi::class)
+class LogNotifier(
+    private val notificationChannel: ReceiveChannel<ProblematicLogs>,
+    private val notifier: NotifierFacade,
+    private val logFiles: LogFiles
+) {
+
+    init {
+        GlobalScope.launch {
+            notificationChannel.consumeEach { logs ->
+                val file = snapshotErrorsToSeparateFile(logs)
+                notifier.notifyLogError("${logs.size} errors on ${logs.instanceName}", "Click to open incident log:\n${logs.logs.last().message}", file)
```

Comment:
the notification text length is very limited. I am not sure that it is a good idea to put here message because it will be badly displayed. maybe display just `Click to open incident log` / with no message, or introduce a check / string shortening here

Reply:
OK

Reply:
Might look ugly, but carries message -> it there was only one error you don't need to click balloon. It is hard to guess what will look good on mac and win.